src/emc/usr_intf/gmoccapy/gmoccapy.py

```diff
@@ -2188,16 +2188,16 @@ def _show_user_message(self, pin, message):
             if pin.get():
                 self.halcomp["messages." + message[2] + "-waiting"] = 1
                 title = "Pin " + message[2] + " message"
-                responce = self.dialogs.show_user_message(self, message[0], title)
+                response = self.dialogs.show_user_message(self, message[0], title)
                 self.halcomp["messages." + message[2] + "-waiting"] = 0
         elif message[1] == "yesnodialog":
             if pin.get():
                 self.halcomp["messages." + message[2] + "-waiting"] = 1
                 self.halcomp["messages." + message[2] + "-response"] = 0
                 title = "Pin " + message[2] + " message"
-                responce = self.dialogs.yesno_dialog(self, message[0], title)
+                response = self.dialogs.yesno_dialog(self, message[0], title)
                 self.halcomp["messages." + message[2] + "-waiting"] = 0
-                self.halcomp["messages." + message[2] + "-response"] = responce
+                self.halcomp["messages." + message[2] + "-response"] = response
```

Comment:
this is probably wrong. Can we rename the variable something other than a typo of `response` ?

Reply:
No, I think removing the typo is not wrong. Why do you think so?

By the way, I have also done these changes (in an open PR): https://github.com/LinuxCNC/linuxcnc/pull/1629/commits/7bf7745e894953afd64c90043e5ff720c3aba8d4 :smile:

Reply:
So this PR is obsolete in that case?

Reply:
Yeah kind of. Sorry.